packages/app/src/components/SQLInlineEditor.tsx

```diff
@@ -149,6 +200,7 @@ export default function SQLInlineEditor({
 
   const updateAutocompleteColumns = useCallback(
     (viewRef: EditorView) => {
+      const currentText = viewRef.state.doc.toString();
```

Comment:
Just a note (and this is probably fine for now, but just calling it out). If a user were to search for a value while in other places in the app (see screenshot), that particular search doesn't show up in the search history. I think it's natural given that we are only capturing true user input, but I wanted to call it out so it's known.

![image](https://github.com/user-attachments/assets/3452f48e-7f5f-4fbe-9e90-056085a6d0d5)

Reply:
good call, I will make a ticket for this.